src/arrayMethodJoin.js

```diff
@@ -4,9 +4,26 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
```

Comment:
Avoid extending the built-in Array prototype. Instead, create a new function that takes array as a parameter.